<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPreparation</class>
 <widget class="QDialog" name="AddPreparation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление препарата</string>
  </property>
  <widget class="QLineEdit" name="leNamePreparation">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>611</width>
     <height>25</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="placeholderText">
    <string>Наименование препарата</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="teditLabelPreparation">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>441</width>
     <height>91</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Описание препарата</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbAdd">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>250</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbCancel">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>250</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>132</y>
     <width>611</width>
     <height>69</height>
    </rect>
   </property>
   <property name="title">
    <string>Поставщики</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QComboBox" name="cboxProviders"/>
    </item>
   </layout>
  </widget>
  <widget class="QSpinBox" name="sboxNumber">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>40</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>40</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Количество</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>100</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Цена за шт.</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="sboxCost">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>100</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="decimals">
    <number>2</number>
   </property>
   <property name="maximum">
    <double>9999.989999999999782</double>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
